nightfall-optimist/src/routes/proposer.mjs

```diff
@@ -542,14 +561,33 @@ router.get('/withdraw', auth, async (req, res, next) => {
  */
 router.post('/payment', auth, async (req, res, next) => {
   const { blockHash } = req.body;
+  const ethAddress = req.app.get('ethAddress');
```

Comment:
Is this way of getting the value better than the one above? maybe we could use the best approach everywhere just to be uniform.